project/src/main/ui/candy-button/candy-button-h4.gd

```diff
@@ -16,51 +16,12 @@ export (Array, Font) var fonts := [
 	preload("res://src/main/ui/candy-button/candy-h6-font.tres"),
 ]
 
-## Gradient which colors the button bright cyan when the button is focused.
-var _gradient_focused: Gradient = preload("res://src/main/ui/candy-button/gradient-focused.tres")
-
-## Gradient which colors the button bright cyan when the button is focused and hovered.
```

Comment:
Preserve this comment.

Reply:
Preserved comment;  `## Gradient which colors the button bright cyan when the button is focused and hovered.`